jina/peapods/compoundpod/__init__.py

```diff
@@ -308,15 +309,15 @@ def wait_start_success(self) -> None:
             self.close()
             raise
 
-    def _enter_replica(self, replica: 'BaseReplica') -> None:
+    def _enter_replica(self, replica: 'Pod') -> None:
         self.replica_list.append(replica)
         self.enter_context(replica)
 
-    def _enter_pea(self, pea: 'BasePea') -> None:
+    def _enter_pea(self, pea: 'Pea') -> None:
```

Comment:
it should not have `_enter_pea` the `CompoundPod` right?

Reply:
yes, changed, thanks

Reply:
It is still there

Reply:
yes one moment, I add the docstrings in the same commit